### ISSUE TITLE
'thumbnail' post-list layout hover state colors don't meet contrast req's

### ISSUE THREAD

**Issue by cjg89** [UCF/Online-Child-Theme#66]
**Describe the bug**

Hover states on the 'thumbnail' post list layout need to be updated to use a bg + text color combo that meets contrast requirements, like we've done recently with other components in Athena.
![screen shot 2019-02-13 at 9 20 22 am](https://user-images.githubusercontent.com/1649216/52718080-9be79980-2f70-11e9-8a4c-dfd998d1b799.png)

**Steps to Reproduce**

1. Hover over a 'thumbnail' post list item on Online, such as one of the links under "Popular Programs" on the homepage
2. Verify hover state is dark gold + white text, which doesn't meet contrast requirements

**Expected Behavior**

Hover state color combo should match other gold/primary component colors used in Athena.

**Possible Fix**

Update the colors used in this component within the theme to match updated navbar link / button hover states in Athena.

**Your environment:**

n/a; all environments

**Additional context**

See screenshot above
